src/RE/FormComponents/TESForm/TESTopicInfo.cpp

```diff
@@ -10,6 +10,13 @@ namespace RE
 		delete next;
 	}
 
+	void TESTopicInfo::ResponseData::PopulateResponseText(TESFile* a_file)
+	{
+			using func_t = decltype(&TESTopicInfo::ResponseData::PopulateResponseText);
+			REL::Relocation<func_t> func{ REL::ID(24985) };
+			func(this, a_file);
```

Comment:
I know this is `void` return and doesn't do anything, but I always write return here just by convention